src/common/asb/Asb.c

```diff
@@ -535,9 +535,13 @@ static const char* g_slapd = "slapd";
 static const char* g_bind9 = "bind9";
 static const char* g_dovecotCore = "dovecot-core";
 static const char* g_auditd = "auditd";
+static const char* g_auditLibs = "audit-libs";
+static const char* g_auditLibsDevel = "audit-libs-devel";
```

Comment:
same as above, is this universal across all distros?

Reply:
No. Read the code changes and you'll see. There are ORed conditions.